src/overmap.cpp

```diff
@@ -946,6 +951,39 @@ void oter_type_t::load( const JsonObject &jo, const std::string_view )
     optional( jo, was_loaded, "connect_group", connect_group, string_reader{} );
     optional( jo, was_loaded, "travel_cost_type", travel_cost_type, oter_travel_cost_type::other );
 
+    if( jo.has_string( "default_map_data" ) ) {
```

Comment:
I prefer these checks to be `has_member`, and let `optional` complain if it's not the format it likes, instead of silently ignoring it.
```suggestion
    if( jo.has_member( "default_map_data" ) ) {
```